modules/storages/config/locales/en.yml

```
@@ -118,8 +118,8 @@ en:
       managed_project_folders_application_password: >
         Copy this value from:
       managed_project_folders_application_password_caption: "Enable automatic managed folders by copying this value from: %{provider_type_link}."
-      new_storage: "Read our %{new_storage_link_text} for more information about this setup."
-      new_storage_link_text: "file storage documentation"
+      new_storage: "Read our documentation on %{file_storage_docs_link} integration for more information."
+      file_storage_docs_link: "setting up a %{provider_name} file storage"
```

I'd like to not split that up because in other languages it is hard to translate the second string without knowing the context of the first string. In German, then usually the wrong case is chosen.

```suggestion
        new_storage: "Read our documentation on <a target='_blank' href='%{provider_link}'>setting up a %{provider_name} file storage</a> integration for more information."
```

---

Makes sense, updating! 👍🏾